docs/react-18-and-react-native.md

```diff
@@ -0,0 +1,119 @@
+---
+id: react-18-and-react-native
+title: React 18 & React Native
+---
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+This page describes how to use React 18 with React Native using the New React Native Architecture.
```

Comment:
```suggestion
This page describes how to use React 18 with React Native using the React Native's New Architecture.
```

I think this reads better and it is more aligned with how we usually refer to the New Architecture.